src/javascript/app/pages/form/barrier-fields.jsx

```diff
@@ -19,8 +19,8 @@ const BarrierFields = ({ form_name, handleAmountChange }) => {
     const barrier_low = Defaults.get(PARAM_NAMES.BARRIER_LOW);
 
     useEffect(() => {
-        setBarrierData((oldData) => ({
-            ...oldData,
+        setBarrierData((old_data) => ({
+            ...old_data,
             ...dataManager.getAllTrades(),
         }));
     }, [hasBarrierChange]);
```

Comment:
`hasBarrierChange` => `has_barrier_change` (line 15, 26)

`barrierData` => other snake_case name (line 14, 28, 53)  
*note: `barrier_data` already existing

Reply:
same for barrierData at line 14